components/cam/src/physics/cam/shoc.F90

```diff
@@ -1751,10 +1751,10 @@ subroutine f0_to_f5_diag_third_shoc_moment(&
   wsec_diff     = w_sec_kc    - w_sec
   tke_diff      = tke_kc      - tke
 
-  f0 = thedz2 * bet2**3 * iso**4 * wthl_sec * &
+  f0 = thedz2 * bfb_cube(bet2) * bfb_pow(iso,4._rtype) * wthl_sec * &
```

Comment:
For 4, we want bfb_quad that evals to bfb_square(bfb_square(x)).

Reply:
There doesn't appear to be a `bfb_quad` in bfb_math.inc. I can add one if you like, or I can change this to `bfb_square(bfb_square(iso))`.

Reply:
I'd say go ahead and add `bfb_quad` if you haven't already.